epub33/reports/a11y-properties-use.md

```diff
@@ -0,0 +1,104 @@
+# EPUB Accessibility 1.1 Metadata Usage Report
+
+## Candidate Recommendation Exit Criteria
+
+The EPUB Working Group intends to exit the Candidate Recommendation stage and submit
+the EPUB Accessibility 1.1 specification for consideration as a W3C Proposed Recommendation
+after documenting implementation of each feature.</p>
+
+For this specification to advance to Proposed Recommendation, it has to be
+proven that metadata defined and required in this specification have sufficient usage by the
+target communities. This metadata falls into two categories:</p>
+
+1. required and recommended schema.org metadata for describing the accessibility
+of publications; and
+2. EPUB-defined metadata properties for reporting aspects of conformance
+
+Usage of these properties means that they are regularly included in the Package Document
+metadata for their EPUB Publications (as appropriate for each title).
+
+## Publisher Implementations
+
+### Schema.org discovery metadata
+
+The following table provides a list of publishers who have stated that they are currently using
```

Comment:
Question: what's a publisher? I self-published a book earlier this year that uses all of these attributes. It will be easy to find one more "publisher" like me ... but I am not, say, Macmillan.

Reply:
Isn't that like asking does a reading system count for testing it doesn't have a significant user base? 😃 

To me, the answer is yes for both cases. We're listing implementations not judging their value so we shouldn't shut anyone out from reporting because they aren't deemed "good enough".

That said, I expect the people reading our implementation reports will judge the value of our implementations so if we only list a few self-publishers then we may not pass.

I'm not too worried about the accessibility metadata (with the exception of the certifierReport property) as GCA has more than two publishers implementing it. The package metadata might get more interesting.